backend/src/modules/lessons/router.js

```diff
@@ -0,0 +1,127 @@
+import { Router } from 'express'
+import { checkSchema, validationResult } from 'express-validator'
+
+import Controller from './controller'
+import verifyUser from '../../middlewares/verifyUser'
+
+const router = Router()
+
+router.use(verifyUser['teacher'])
```

Comment:
use of wrong middleware. We have another middleware called checkRoles.js.
Example of using checkRoles.js:
router.use(checkRoles(['array', 'of', 'roles', 'that', 'has', 'access', 'to', 'this', 'module', 'or', 'route']))